README.md

```diff
@@ -25,37 +23,69 @@ Then start the server with gulp.
 gulp
 ```
 
+You can also use gulp with a local .env file for environmental variables.
+
+```
+npm run start-local
+```
+
+See [package.json](package.json)'s `scripts` section for more details.
+
+### Local .env file configurations
+
+Using a local .env file, you can set environmental vairables to emulate
+different running conditions.
+
+*   **PORT** Specify the default port that the server runs at.
+*   **GOOGLE_SITE_VERIFICATION** Set a value for
+    [HTML Meta tag site verification](https://support.google.com/webmasters/answer/9008080?hl=en)
+*   **PROXY_URL** Circumvent https requirement for non-localhost testing
+    environs.
+*   **HOST** Can be set to `0.0.0.0` for
+    [non-localhost routing using gulp](https://github.com/schickling/gulp-webserver#why-cant-i-reach-the-server-from-the-network).
+*   **GSI_CLIENT_ID** Can be set to specify a custom
+    [Google Sign-in Client Id](https://developers.google.com/identity/sign-in/web/sign-in).
+
+A sample `.env` file:
+
+```
+PORT=8080
+GOOGLE_SITE_VERIFICATION=abc123
+PROXY_URL=test-url.local
+HOST=0.0.0.0
+GSI_CLIENT_ID=520465458218-e9vp957krfk2r0i4ejeh6aklqm7c25p4.apps.googleusercontent.com
+```
+
 ## How to update swg.js for local mode
 
 After the server starts to run in localhost, you can select `local` mode to
 inspect swg.js served from ./swgjs directory. The way to update ./swgjs is
 
```

Comment:
Sorry about the below formatting changes, the editor lint rules went overboard

Reply:
No worries! Could you use `prettier` for formatting this markdown file?

Reply:
Will do shortly.

Reply:
Thanks!